osu.Game/Overlays/Dashboard/CurrentlyPlayingDisplay.cs

```diff
@@ -130,6 +130,9 @@ protected override void OnFocus(FocusEvent e)
                                 if (!playingUsers.Contains(user.Id))
                                     return;
 
+                                // TODO: remove this once online state is being updated more correctly.
+                                user.IsOnline = true;
```

Comment:
@bdach please check on this. i can split out into another PR for better visibility but until we complete https://github.com/ppy/osu/issues/12635 feels like we need this.

Reply:
The only problem I can see with this is that this is mutating an instance that comes from the user cache, so this is "polluting" the cache, in a way. But we're going to have to figure out what _proper_ invalidation there would look like, and it's not like setting this is _actively_ wrong or anything, so I'm willing to look away temporarily...